### PR TITLE
Remove deprecated AWS Marketplace builder

### PR DESCRIPTION
**Pull request by michal-harish**

### CODE REVIEW

File: build/aws-marketplace/cloudbuild.yaml [hunk 1]
```diff
@@ -19,6 +19,6 @@ steps:
       - '-ec'
       - |
         source ~/.heroic-ecr-secret && \
-        docker tag nakama-awsmp:${TAG#v} $_MAP_ECR_REPOSITORY:$TAG
+        docker tag nakama-awsmp:${TAG#v} $_MAP_ECR_REPOSITORY:${TAG#v}-m1
```
Comment (mofirouz) [inline build/aws-marketplace/cloudbuild.yaml:22]: What's this change?

Comment (michal-harish) [reply build/aws-marketplace/cloudbuild.yaml:22]: That's versioning of the images since we can't replace them in the mp ecr.

Comment (mofirouz) [reply build/aws-marketplace/cloudbuild.yaml:22]: I think we leave the old images as they are - and we just publish Nakama images that don't exist yet.

Comment (mofirouz) [reply build/aws-marketplace/cloudbuild.yaml:22]: Alternatively we use the `r` suffix as we've used everywhere else before that stands for `revision`. I don't know what `M` stands for here?

Comment (mofirouz) [reply build/aws-marketplace/cloudbuild.yaml:22]: Ok that's fine, let's use `-m` suffix :)

**Pull request merged**